lib/game_teams_manager.rb

```diff
@@ -56,28 +56,28 @@ def teams_sort_by_average_goal
       team_average_goals(team.team_id)
     end
   end
-end
 
-#   def find_all_away_teams
-#      @game_teams_array.find_all do |gameteam|
-#       gameteam.hoa == "away"
-#     end
-#   end
+  def find_all_away_teams
```

Comment:
Love how simple this method is. This approach (as opposed to our previous) is much better don't you think?